less/_defaults/_colors.less

```diff
@@ -55,6 +55,14 @@
 @btn-color-hover: darken(@btn-color, 10%);
 @btn-color-inverted-hover: @btn-color-inverted;
 
+// Button icon
+// --------------------------------------------------
+@btn-icon-color: @greyDark;
```

Comment:
Could button icon colours inherit from `@nav-icon` by default? Button icons tend to be used for navigation elements next/previous etc so I often inherit from `@nav-icon` for consistency.

```
@btn-icon-color: @nav-icon;
@btn-icon-color-inverted: @nav-icon-inverted;

@btn-icon-color-hover: @nav-icon-hover;
@btn-icon-color-inverted-hover: @nav-icon-inverted-hover;
```
schema _default_ and _help_ will need amending to reflect.